substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/NativeImage.java

```diff
@@ -519,6 +523,44 @@ public Path getBuilderInspectServerPath() {
             return builderJavaArgs;
         }
 
+        private void setHasGraalModules() {
```

Comment:
I really wanted to get rid of stunts like this in the driver (as you can see in the removing of `getBuilderModuleNames` in https://github.com/oracle/graal/pull/3763).

Do we still need this if we solve the `jdk.scripting.nashorn` problems as suggested above?
Or do you have other means of knowing that this is not our GraalVM (i.e. different value for e.g. `com.oracle.svm.driver.NativeImage#graalvmVersion` or `com.oracle.svm.driver.NativeImage#graalvmConfig`)

Reply:
As we discussed on slack, this is no longer needed and we are using extra modules on the module path to get around the missing module error when building ni modules, like the native-image-agent.